src/LanguageServer/Protocol/Handler/Diagnostics/DiagnosticSources/IDiagnosticSource.cs

```diff
@@ -29,7 +29,6 @@ internal interface IDiagnosticSource
     /// errors from the past, we do want them to be superseded by a more recent live run, or a more recent build from
     /// another source.
     /// </summary>
-    bool IsLiveSource();
```

Comment:
It feels like the doc comment also needs to be moved or deleted. Perhaps we could ensure the definition of the Build diagnostic tag has info similar to this.